src/iconsManifest/supportedExtensions.ts

```diff
@@ -1856,7 +1856,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'helm',
-      extensions: [],
+      extensions: ['Chart.lock', 'Chart.yaml'],
```

Comment:
Put the filenames in lowercase, otherwise it won't work. You'll  have to add `filename: true`, too.